### PR TITLE
Fix wrong dropped file view

### LINKED ISSUE #14951: Enhancement: When in side-by-side mode, when dragging a file to NP++, open file on side file dropped
### Description of the Issue
With NP++ displaying in side-by-side mode, open dragged file on side where dropped

### Steps to Reproduce the Issue
1. Activate side-by-side display
2. Have a file on both sides
3. Drag a file from file explorer to side not most recently active

### Expected Behavior
Dragged file opens on side it was dropped

### Actual Behavior
The file opens on the side that was most recently active

### Debug Information
Notepad++ v8.6.4   (64-bit)
Build time : Feb 20 2024 - 00:12:59
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line :
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit)
OS Version : 22H2
OS Build : 19045.4170
Current ANSI codepage : 1252
Plugins :
    DSpellCheck (1.5)
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)

### PR DESCRIPTION
**Pull request by xomx**
Fix #14951 .

### CODE REVIEW

File: PowerEditor/src/Notepad_plus.cpp [hunk 1]
```diff
@@ -4281,7 +4281,7 @@ void Notepad_plus::dropFiles(HDROP hdrop)
 		HWND hWin = ::ChildWindowFromPointEx(_pPublicInterface->getHSelf(), p, CWP_SKIPINVISIBLE);
 		if (!hWin) return;
 
-		if ((_subEditView.getHSelf() == hWin) || (_subDocTab.getHSelf() == hWin) || currentView() == SUB_VIEW)
+		if ((_subEditView.getHSelf() == hWin) || (_subDocTab.getHSelf() == hWin))
```
Comment (donho) [inline PowerEditor/src/Notepad_plus.cpp:4284]: It kind of revert https://github.com/notepad-plus-plus/notepad-plus-plus/commit/5f13c847191b38313d456fd69f575c5d84137ea0, which is supposed to fix #14354

**Review by donho — CHANGES_REQUESTED** [review]
Could you find another solution to fix both #14354 & #14951?

File: PowerEditor/src/Notepad_plus.cpp (hunk 1 above)
Comment (xomx) [reply PowerEditor/src/Notepad_plus.cpp]: @donho 
You are right, my previous fix was not universal.

**Comment by xomx** [comment]
@donho 
> Could you find another solution to fix both https://github.com/notepad-plus-plus/notepad-plus-plus/issues/14354 & https://github.com/notepad-plus-plus/notepad-plus-plus/issues/14951?

Ok, I have fixed both.
But kudos here goes to @Yaron10, who suggested [here](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/14354#issuecomment-1811637865) the correct solution before, but you chose a different but incorrect solution.

**Review by donho — APPROVED** [review]

**Pull request closed without merging**